### PR TITLE
feat(nodejs): Deprecate `frameContextLines` option and move to integration

### PR DESCRIPTION
**Pull request by timfish**
These are docs updates to cover the changes in this PR:
https://github.com/getsentry/sentry-javascript/pull/3729

- Adds a new `ContextLines` integration which has a `frameContextLines` option
- Marks `frameContextLines` in node options as deprecated and links to the integration

### CODE REVIEW

File: src/platforms/node/common/configuration/integrations/default-integrations.mdx [hunk 1]
```diff
@@ -100,6 +100,21 @@ Available options:
 }
 ```
 
+### ContextLines
+
+_Import name: `Sentry.Integrations.ContextLines`_
```
Comment (AbhiPrasad) [inline src/platforms/node/common/configuration/integrations/default-integrations.mdx:105]: We'll have to add a note, new in version X.

```suggestion
_Import name: `Sentry.Integrations.ContextLines`_

_(Available in version 6.X.X and above)_
```

Comment (timfish) [reply src/platforms/node/common/configuration/integrations/default-integrations.mdx:105]: So the JavaScript PR gets merged, add version here and then release together or something like that?

Comment (AbhiPrasad) [reply src/platforms/node/common/configuration/integrations/default-integrations.mdx:105]: Yup exactly!

**Comment by timfish** [comment]
I added the version number but now it appears to be failing on broken links for what I have added!

**Review by AbhiPrasad — APPROVED** [review]

**Comment by AbhiPrasad** [comment]
:ohno: there's some anchor issues. Maybe `configuration/integrations/default-integrations/#contextlines` doesn't exist for all node platforms?

**Comment by AbhiPrasad** [comment]
> Perhaps the link checker doesn't respect conditional sections

It should since we run it on the built assets. @imatwawana, have you come across this before?

File: src/platforms/common/configuration/options.mdx [hunk 2]
```diff
@@ -306,6 +306,10 @@ The number of context lines for each frame in the stacktrace when loading a file
 
 The number of context lines for each frame when loading a file.
 
+#### Deprecated
+
+`frameContextLines` has moved to the <PlatformLink to="/configuration/integrations/default-integrations/#contextlines/">`ContextLines` integration</PlatformLink>.
```
Comment (imatwawana) [inline src/platforms/common/configuration/options.mdx:311]:
```suggestion
`frameContextLines` has moved to the <PlatformLink to="/configuration/integrations/default-integrations/#contextlines">`ContextLines` integration</PlatformLink>.
```

**Comment by imatwawana** [comment]
> > Perhaps the link checker doesn't respect conditional sections
> 
> It should since we run it on the built assets. @imatwawana, have you come across this before?

I'm going to try a couple things to see if it fixes the issue.

**Comment by timfish** [comment]
Oh nice. Thank you!

**Pull request merged**